src/main/java/frc/robot/OperatorInterface.java

```diff
@@ -60,8 +60,19 @@ public double getTurnCmd() {
      * the joysticks.
      */
     public void update() {
-        this._driveCmd = -this._pilotJoy.getRawAxis(RobotConstants.AXIS_DRIVE);
-        this._turnCmd = -this._pilotJoy.getRawAxis(RobotConstants.AXIS_TURN);
+        if(Math.abs(-this._pilotJoy.getRawAxis(RobotConstants.AXIS_DRIVE)) < RobotConstants.DeadBand){
+            this._driveCmd = (0.0);
+        }
+        else{
+            this._driveCmd = -this._pilotJoy.getRawAxis(RobotConstants.AXIS_DRIVE);
+        }
```

Comment:
Read the value before you enter the if statement. Otherwise, the reading you get in line 63 can be different from the reading you get in line 67.

You can keep the old code - i.e.
```java
this._driveCmd = -this._pilotJoy.getRawAxis(RobotConstants.AXIS_DRIVE);
this._turnCmd = -this._pilotJoy.getRawAxis(RobotConstants.AXIS_TURN);
```
and then just feed those values into the if-else statements